mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -3253,6 +3253,106 @@ pub async fn enable_eth_with_tokens_v2(
     }
 }
 
+async fn init_erc20_token(
+    mm: &MarketMakerIt,
+    ticker: &str,
+    protocol: Option<Json>,
+    is_custom: bool,
+    path_to_address: Option<HDAccountAddressId>,
+) -> Result<(StatusCode, Json), Json> {
+    let (status, response, _) = mm.rpc(&json!({
+        "userpass": mm.userpass,
+        "method": "task::enable_erc20::init",
+        "mmrpc": "2.0",
+        "params": {
+            "ticker": ticker,
+            "protocol": protocol,
+            "activation_params": {
+                "is_custom": is_custom,
```

Comment:
`protocol` is only valid with `is_custom`, and `is_custom` could only be set with `protocol`? did i get that right?

i feel like we would be better of including the `protocol` inside `activation_params` and never flagging the custom thing, we will know it's custom when we don't find it in the config.

Reply:
> i feel like we would be better of including the protocol inside activation_params

`activation_params` is coin type/protocol specific, while custom token support should be implemented (gradually) for all coins/protocols that support tokens. It wouldn't be logical to include protocol information in protocol-specific params.

I opted for another solution here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2141/commits/e9b6dc79b20d65710a28818bcce618bb56aac716 . I avoided this before as I didn't want to pass `is_custom` parameter through functions but this is better than a redundant parameter in requests.